test/032-db-backup.tf

```diff
@@ -21,4 +21,11 @@ module "backup" {
   aws_backup_notification_events   = [""]
   backup_sns_email                 = ""
   delete_recovery_point_after_days = 7
+  enable_s3_to_b2_sync             = false
+  b2_application_key_id            = ""
+  b2_application_key               = ""
+  b2_bucket                        = ""
+  rclone_arguments                 = "--transfers 4 --checkers 8"
+  b2_path                          = ""
+  sync_schedule                    = null
```

Comment:
This is "null", but the data type is "string" above. Are strings nullable in Terraform?

Reply:
Good question. All variables are nullable by default in Terraform. And as far as I can recall, we have never set `nullable = false` on any of our variables.

Since this test file only acts as a basic syntax check, this is legitimate. If we did a more realistic test, it would probably have to be a valid schedule.